src/apme_engine/remediation/abbenay_client_factory.py

```
@@ -0,0 +1,221 @@
+from __future__ import annotations
+
+import inspect
+import logging
+import os
+from dataclasses import dataclass
+
+import grpc
+
+logger = logging.getLogger(__name__)
+
+_DEFAULT_CA_CERT = "/tmp/abbenay-run/abbenay/tls/ca.crt"
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/abbenay-run/abbenay/tls/ca.crt"

(S108)